symplyphysics/laws/quantum_mechanics/harmonic_oscillator/wave_eigenfunctions.py

```diff
@@ -0,0 +1,65 @@
+from sympy import Eq, Rational, sqrt, exp, factorial, pi
+from sympy.functions.special.polynomials import hermite
+from sympy.physics.units import hbar
+from symplyphysics import (
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    units,
+    dimensionless,
+    angle_type,
+    symbols,
+    clone_symbol,
+)
+
+# Description
+## The time-independent Schrödinger equation for the harmonic potential can be solved using the
+## spectral method in the basis of Hermite functions.
+
+# Law: psi_n(x) = 1 / sqrt(2**n * factorial(n))
+#               * ((m * w) / (pi * hbar))**(1 / 4)
+#               * exp(-1 * (m * w) / (2 * hbar) * x**2)
+#               * H_n(sqrt((m * w) / hbar) * x)
+## psi_n - n-th wave eigenfunction
+## n - mode number (n = 0, 1, 2, ...)
+## m - mass of oscillating quantum particle
+## w - angular frequency of quantum oscillator
+## hbar - reduced Planck constant
+## H_n - n-th (physicists') Hermite polynomial
+
+wave_function = Symbol("wave_function", 1 / sqrt(units.length))
+mode_number = Symbol("mode_number", dimensionless, integer=True, nonnegative=True)
+oscillator_mass = clone_symbol(symbols.basic.mass, "oscillator_mass")
+angular_frequency = Symbol("angular_frequency", angle_type / units.time)
+position = Symbol("position", units.length)
+
+law = Eq(
+    wave_function,
+    (1 / (sqrt(2**mode_number * factorial(mode_number))))
+    * ((oscillator_mass * angular_frequency) / (pi * hbar))**Rational(1, 4)
```

Comment:
You can put `oscillator_mass * angular_frequency` to a separate expression.

Reply:
I'm not really sure what to call it so it's better to leave it the original way.

Reply:
Sometimes we simply call them `expression1`, `expression2`. It shortens formula a bit and nothing more.

Reply:
I believe in the case of this law it's better to leave it explicit and not make it more obscure than it should be.